pe/pe.py

```diff
@@ -650,6 +680,11 @@ def add_headers(self):
                 )
                 res.add_subsection(heur_section)
 
+        # Inspired by https://github.com/viper-framework/viper-modules/blob/00ee6cd2b2ad4ed278279ca9e383e48bc23a2555/lief.py#L314
+        binary_type = lief.PE.get_type(self.request.file_path)
```

Comment:
I'm not certain the code will reach this part if LIEF is not able to determine the binary type. It may be equivalent to current heuristic 7? We'll see if we never see this ResultSection.